templates/modify-vm-template/manifests/modify-vm-template.yaml

```diff
@@ -51,11 +51,11 @@ spec:
       name: vmAnnotations
       type: array
       default: []
-    - description: 'VM disks in json format, replace vm disk if same name, otherwise new disk is appended. Eg ["{\"name\": \"test\", \"cdrom\": {\"bus\": \"sata\"}}", "{\"name\": \"disk2\"}"]'
+    - description: 'VM disks in json format, replace vm disk if same name, otherwise new disk is appended. Eg [{"name": "test", "cdrom": {"bus": "sata"}}, {"name": "disk2"}]'
```

Comment:
does the disk have to be surrounded by `'` for this to work or the current state is enought? (context https://github.com/kubevirt/kubevirt-tekton-tasks/pull/101#discussion_r742584288)

Reply:
this depends how you are working with the task. If you e.g. pass it via UI, you don't need any quotation marks, but if you are working with CLI, you need `'`

Reply:
ok then